cluster/charts/rook-ceph-cluster/templates/deployment.yaml

```diff
@@ -24,9 +24,10 @@ spec:
       containers:
         - name: rook-ceph-tools
           image: {{ .Values.toolbox.image }}
-          command: ["/tini"]
-          args: ["-g", "--", "/usr/local/bin/toolbox.sh"]
+          command: ["/bin/bash"]
+          args: ["-cm", "/usr/local/bin/toolbox.sh"]
```

Comment:
-m basically replaces tini in this case?

Reply:
Not all of it but enough for our use case :-)

Reply:
Shouldn't `-c` be the last arg?

Should we separate these args so it's easier to understand?